### PR TITLE
switch GCR -> Artifact-Registry

### PR DESCRIPTION
**Pull request by ccwienk**
GCR has been deprecated [0] in favour of Artifact-Registry.

Thus, change push-targets for OCI-Images:

- europe-docker.pkg.dev/gardener-project/snapshots for snapshots
- europe-docker.pkg.dev/gardener-project/releases for releases
- europe-docker.pkg.dev/gardener-project/public for combined view of snapshots + releases

[0]
https://cloud.google.com/artifact-registry/docs/transition/transition-from-gcr

**Release note**:

```breaking operator
Change OCI Image Registry from GCR (`eu.gcr.io/gardener-project`) to Artifact-Registry (`europe-docker.pkg.dev/gardener-project/releases`). Users should update their references.

```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,9 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-PROJECT          := ingress-default-backend
+PROJECT          := gardener/ingress-default-backend
 VERSION          := $(shell cat VERSION)
-REGISTRY         := eu.gcr.io/gardener-project/gardener
+REGISTRY         := europe-docker.pkg.dev/gardener-project/public
```
Comment (ScheererJ) [inline Makefile:17]: Please move `gardener` to the `REGISTRY` as it was before.

**Review by ScheererJ — CHANGES_REQUESTED** [review]
Why is the additional ocm repository mapping not required for the `head-update` similar to the corresponding changes in other repositories?

File: Makefile (hunk 1 above)
Comment (ccwienk) [reply Makefile:17]: done

**Comment by ccwienk** [comment]
> Why is the additional ocm repository mapping not required for the `head-update` similar to the corresponding changes in other repositories?

For `head-update`, the additional ocm-repository is only required for "draft"-trait (which collects release notes / updates release-drafts). in this case, this trait is not present, so the additional ocm-repository is not required (it would have no effect to configure it).

**Review by ScheererJ — APPROVED** [review]
/lgtm
/approve

**Pull request merged**